### ISSUE TITLE
[Activity] Not found page text should be different for my strategies and individual strategy vs the explorer

### ISSUE THREAD

**Issue by tiagofilipenunes** [bancorprotocol/carbon-app#1347]
The current text for when there is no activity found in my strategy, individual strategies and the explorer is:

```
We couldn't find any activities
Try entering a different wallet address or choose a different token pair.
```

This text only makes sense in the context of the explorer page.

**Comment by ashachaf** [comment]
explorer page:
- pair search
```
We couldn't find any strategies
Try a different token pair or reset your filters.
```
- wallet search
```
We couldn't find any strategies
Try entering a different wallet address or reset your filters.
```

my strategies:

strategy page:
```
Strategy not found
The strategy you are looking for does not exist or has been deleted.
```

**Comment by tiagofilipenunes** [comment]
Fixed as part of #1426
